homekit.html

```diff
@@ -56,7 +60,7 @@ <h3 id="toc_5">Service</h3>
       <p>The Service node represents the single device you want to control or query.</p>
       <p>Every service node creates its own HAP accessory to keep things simple.</p>
       <ul>
-          <li><strong>Topic</strong>: An optional property that can be configured in the node or, if left blank, can be set by <code>msg.topic</code>.</li>
+          <li><strong>Topic</strong>: An optional property that can be configured in the node or, if left blank, can be set by <code>msg.topic</code>. If <em>Filter on Topic</em> is selected <code>msg.topic</code> of incoming messages must match the configured value for the message to be accepted.</li>
```

Comment:
Additional note to the suggestion in `HAPServiceNode.js`:

```suggestion
          <li><strong>Topic</strong>: An optional property that can be configured in the node or, if left blank, can be set by <code>msg.topic</code>. If <em>Filter on Topic</em> is selected <code>msg.topic</code> of incoming messages must match the configured value for the message to be accepted. If <em>Filter on Topic</em> is selected and no <em>Topic</em> is set on the node, then <code>msg.topic</code> must match the nodes <em>Name</em></li>
```

Reply:
Like this addition @flic?